lib/parsers/tiff_parser.rb

```diff
@@ -54,8 +54,9 @@ def cr2?(io)
 
   # Similar to how exiftool determines the image type as ARW, we are implementing a check here
   # https://github.com/exiftool/exiftool/blob/e969456372fbaf4b980fea8bb094d71033ac8bf7/lib/Image/ExifTool/Exif.pm#L929
+  # Change in version 1.7.0 to use start_with for make matching due to some sample images having a padded make
```

Comment:
Not sure whether we need this comment 🤔

Reply:
👍 I'll remove it. Wasn't sure if we wanted to indicate existing functionality changes back to versions but I doubt it's very useful.